apps/src/componentLibrary/StylizedBaseDialog.jsx

```diff
@@ -101,7 +102,7 @@ export default function StylizedBaseDialog(props) {
           ...(styles.body[props.type] || {})
         }}
       >
-        {props.body}
+        {props.body ? props.body : props.children}
```

Comment:
It's a pattern that @jmkulwik suggested -- it does make testing easier because you can assert on `CodeReviewGroupsManager` in a shallow/isolate-components rendered component, since it is rendered directly by `ManageCodeReviewGroups` (rather than passed as a prop to `StylizedBaseDialog`).

Reply:
I think it's particularly useful in this case if we want to access `groups` and `setGroups` in tests, because I don't know how you would do so otherwise. There's something called `dive()` in Enzyme that lets you access children of children in shallow rendered components, but isolateComponents has no equivalent.